### ISSUE TITLE
Add ambiguity checking to System Order of Execution section

### ISSUE THREAD

**Issue by alice-i-cecile** [bevy-cheatbook/bevy-cheatbook#44]
Useful to have a quick lookup of the incantation. 

Obviously needs caveats on when to care, and how to interpret.

**Comment by jamadazi** [comment]
I don't think this is an urgent priority. I'll try to add it soon, sometime after merging #33 .
